.travis.yml

```diff
@@ -16,65 +16,43 @@ before_install:
   - docker pull thesofproject/sof && docker tag thesofproject/sof sof
 
 env:
-  - PLATFORM=byt
-  - PLATFORM=cht
-  - PLATFORM=bdw
-  - PLATFORM=hsw
-  - PLATFORM=apl
-  - PLATFORM=skl
-  - PLATFORM=kbl
-  - PLATFORM=cnl
   - PLATFORM=sue
-  - PLATFORM=icl
   - PLATFORM=jsl
-  - PLATFORM=imx8
-  - PLATFORM=imx8x
-  - PLATFORM=imx8m
 
 script: ./scripts/docker-run.sh ./scripts/xtensa-build-all.sh -j $PLATFORM
 
 jobs:
   include:
+
     - stage: test
       script: ./scripts/docker-run.sh ./scripts/build-tools.sh
       env: PLATFORM=tools
+
     - stage: test
       script: ./scripts/docker-run.sh ./scripts/host-build-all.sh -l
       env: PLATFORM=host
       # Matrix hack: Declare the same stage name multiple times to test multiple
       # versions. Use a YAML alias to prevent redundancy.
+
     - &qemuboottest
       stage: qemutest
       script:
         - sed -i $(($(grep "config HAVE_AGENT" -n src/platform/Kconfig | cut -f1 -d:)+2))"s/default y/default n/" src/platform/Kconfig
         - ./scripts/docker-run.sh ./scripts/xtensa-build-all.sh -r -j $PLATFORM
         - ./scripts/docker-qemu.sh ../sof.git/scripts/qemu-check.sh $PLATFORM
-      env: PLATFORM=byt
+      env: PLATFORM='byt cht'
       before_install:
         - docker pull thesofproject/sof && docker tag thesofproject/sof sof
         - docker pull thesofproject/sofqemu && docker tag thesofproject/sofqemu sofqemu
     - <<: *qemuboottest
-      env: PLATFORM=cht
-    - <<: *qemuboottest
-      env: PLATFORM=bdw
-    - <<: *qemuboottest
-      env: PLATFORM=hsw
-    - <<: *qemuboottest
-      env: PLATFORM=apl
+      env: PLATFORM='bdw hsw'
     - <<: *qemuboottest
-      env: PLATFORM=skl
+      env: PLATFORM='apl skl kbl'
     - <<: *qemuboottest
-      env: PLATFORM=kbl
+      env: PLATFORM='cnl icl'
     - <<: *qemuboottest
-      env: PLATFORM=cnl
-    - <<: *qemuboottest
-      env: PLATFORM=icl
-    - <<: *qemuboottest
-      env: PLATFORM=imx8
-    - <<: *qemuboottest
-      env: PLATFORM=imx8x
-    - <<: *qemuboottest
-      env: PLATFORM=imx8m
+      env: PLATFORM='imx8 imx8x imx8m'
+
```

Comment:
Testtbench is showing platform = sue which is wrong as it runs on host.

Reply:
For now it is showing "byt" because of the way Travis matrix works, also wrong.

I'll see if I can do something about this.

Reply:
Done with a `name:`.

Note Travis never shows both a `name:` and `env:` on the same line

Reply:
Done.